src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -43,6 +43,7 @@ var addCommentHandler = function(e) {
 	                    var newCommentRow = addFormRow.prev();
 	                    newCommentRow.find("form[class*='responseCommentEditForm'] > div > a[id*='button_save_comment_for_edit']").click(editCommentHandler);
 	                    newCommentRow.find("form[class*='responseCommentDeleteForm'] > a").click(deleteCommentHandler);
+	                    registerCheckboxEventForVisibilityOptions();
```

Comment:
yes. you are right. but registerResponseCommentsEvent is just another source of bug

```
    $("form[class*='responseCommentAddForm'] > div > a").click(addCommentHandler);
    $("form[class*='responseCommentEditForm'] > div > a").click(editCommentHandler);
    $("form[class*='responseCommentDeleteForm'] > a").click(deleteCommentHandler);
```